src/app/clusters/messages-server/messages-delegate.h

```diff
@@ -31,17 +31,23 @@ namespace Messages {
 
 using MessageResponseOption = chip::app::Clusters::Messages::Structs::MessageResponseOptionStruct::Type;
 
+constexpr static size_t kMessageIdLength               = 16;
+constexpr static size_t kMessageTextLengthMax          = 256;
+constexpr static size_t kMessageMaxOptionCount         = 4;
+constexpr static size_t kMessageResponseIdMin          = 1;
+constexpr static size_t kMessageResponseLabelMaxLength = 32;
+
 class Delegate
 {
 public:
     // Commands
-    virtual void
+    virtual CHIP_ERROR
     HandlePresentMessagesRequest(const ByteSpan & messageId, const MessagePriorityEnum & priority,
```

Comment:
not for this PR, however `const MessagePriorityEnum & priority` should probably be `MessagePriorityEnum  priority`. We should not pass basic types by reference.

Reply:
I will keep this in mind (although this predates this PR)